### PR TITLE
Store: WooCommerce: Add order creation date to the order details

### LINKED ISSUE #16381: Store: WooCommerce: Order details should include order datetime
When viewing an order (at /store/order/{domain}/{orderID} ), you cannot tell when the order was created.

### PR DESCRIPTION
**Pull request by allendav**
Fixes #16381 

To test, go look at an individual order. The time is, like that for posts, presented in the browser's timzeone.

screen shot 2017-07-31 at 5 25 40 pm

cc @kellychoffman @jameskoster for design review

### CODE REVIEW

File: client/extensions/woocommerce/app/order/order-created.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import Gridicon from 'gridicons';
+import { localize, moment } from 'i18n-calypso';
```
Comment (ryelle) [inline client/extensions/woocommerce/app/order/order-created.js]: `moment` is also [passed as a prop](https://github.com/Automattic/i18n-calypso#localize) from calling `localize`, like `translate`, so you don't need to import it here.

Comment (allendav) [inline client/extensions/woocommerce/app/order/order-created.js]: TIL - ty!

**Comment by allendav** [comment]
@ryelle wrote

> I'm not sure about the placement between total and refund, it feels like it breaks that flow.

Yeah, this was a hard one to think of the best place to put it. My first inclination was immediately below the "Order XXX Details" label in the Section Heading, but that's bending the Section Heading design a fair bit. I also considered adding it to the left of the "Total" but on narrow devices that will could get cramped/confusing.

So, since the "refund" and "fulfillment" "rows" seem quasi-chronological (i.e. I could see adding timestamps to them at some point too) I settled on the position you see in this PR - as a "row" before the refund. But I get your point that this separates the total and refund visually a bit.

How about I fix the import business but since @kellychoffman and @jameskoster are out for the moment we proceed with this and then if they want to move it I'll gladly move it in a subsequent PR?

**Comment by ryelle** [comment]
Sure, that works for me. I'll add an issue to make sure it gets seen to 🙂

File: client/extensions/woocommerce/app/order/order-created.js (hunk 1 above)
Comment (allendav) [inline client/extensions/woocommerce/app/order/order-created.js]: Fixed in https://github.com/Automattic/wp-calypso/pull/16777/commits/6eb94069be9491071840c11c9e1607062f0d1861

**Pull request merged**